### ISSUE TITLE
Add New Board Member - Katy Yaroslavsky

### ISSUE THREAD

**Issue by shrayshray** [Metro-Records/la-metro-councilmatic#953]
Katy Yaroslavsky is an Appointee of Mayor of the City of Los Angeles. She has been added to the Regular Board Meeting in Legistar starting 3/15/2023. We are waiting on her Committee appointments. I will add her bio and headshot to this issue once we receive them.

**Comment by hancush** [comment]
@shrayshray We've added her post. The site is ready for you when you're ready to add her bio and headshot!

**Comment by shrayshray** [comment]
@hancush thank you! OMG I just updated my first Board Member! All looks good on her page, but the smaller headshot on her entry in the Board Members list is still blank. Will this update automatically eventually, or is it a separate task? 
![image](https://user-images.githubusercontent.com/28063988/226068571-29a9bf50-f98c-4504-b0f4-c8c669f20d45.png)
I attached her headshot as well in case it need to be added separately for the Board Members page.

![Councilwoman Yaroslavsky - Portrait (Jan 2023)](https://user-images.githubusercontent.com/28063988/226068105-639897e7-009e-4af4-9aa7-7027651c7752.jpg)
